roles/pulp_content_guard_rbac/README.md

```diff
@@ -0,0 +1,41 @@
+pulp_content_guard_rbac
+================
+
+This role creates and deletes Pulp RBAC content guards using the Pulp API
+
+Role variables
+--------------
+
+* `pulp_url`: URL of Pulp server. Default is `https://localhost:8080`
+* `pulp_admin_username`: Username used to access Pulp server. Default is `admin`
+* `pulp_admin_password`: Password used to access Pulp server. Default is unset
```

Comment:
Is it necessarily admin? The pulp_content_guard role just has pulp_username/pulp_password